internal/ingress/controller/store/store.go

```diff
@@ -773,3 +835,20 @@ func (s k8sStore) Run(stopCh chan struct{}) {
 		go wait.Until(s.checkSSLChainIssues, 60*time.Second, stopCh)
 	}
 }
+
+// ListControllerPods returns a list of ingress-nginx controller Pods
+func (s k8sStore) ListControllerPods() []*corev1.Pod {
```

Comment:
Can you make this clear that it returns only `Running` pods? Something like `ListRunningControllerPods`